Mathlib/Algebra/GroupWithZero/Defs.lean

```diff
@@ -259,6 +260,21 @@ theorem mul_eq_zero_of_left {a : M₀} (h : a = 0) (b : M₀) : a * b = 0 := h.s
 
 theorem mul_eq_zero_of_right (a : M₀) {b : M₀} (h : b = 0) : a * b = 0 := h.symm ▸ mul_zero a
 
+lemma noZeroDivisors_iff_eq_zero_of_mul_left :
```

Comment:
```suggestion
lemma noZeroDivisors_iff_right_eq_zero_of_mul :
```
is clearer IMO...

Reply:
I got the idea of the name from [IsLeftRegular.mul_left_eq_zero_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/GroupWithZero/Regular.html#IsLeftRegular.mul_left_eq_zero_iff), but since the RHS here isn't iff I agree yours is clearer.